benchmark/spawn-cost.sh

```
@@ -0,0 +1,374 @@
+        const result = Math.random() * 100;
+      });
+
+      // Keep alive
+      process.stdin.resume();
+    " &
+    pids+=($!)
+  done
+
+  sleep 1 # Allow agents to start
+
+  local start_time=$(date +%s%3N)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)